packages/genesys-spark-components/src/components/beta/gux-side-panel/gux-modal-side-panel.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Listen,
+  Method,
+  Prop,
+  Watch
+} from '@stencil/core';
+import { GuxSidePanelSize } from './gux-side-panel.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+import { hasSlot } from '@utils/dom/has-slot';
+import { trackComponent } from '@utils/tracking/usage';
+
+/**
+ * @heading - The heading of the side panel
+ * @description - Optional description of the side panel
+ * @content - The content of the side panel
+ * @footer - The footer of the side panel
+ */
```

Comment:
I think maybe using the @slot annotation syntax might be better here but also, I don't see any readme being built from this?